.github/workflows/build-and-test-BASE.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest]
-        framework: [net452, net462, net472, net480, net481, netcoreapp3.1, net6.0, net7.0]
+        framework: [net452, net462, net472, net480, net481, netcoreapp3.1, net6.0, net7.0, net8.0]
```

Comment:
can we remove net452 ?

Reply:
unfortunately, no. some internal teams still depend on this.